entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+python3 -m http.server 8080 --directory /iglu-schemas &
```

Comment:
I suppose this would fail if the user runs image with the `--network=host` option, and if port 8080 is already open on the host machine?

It is a legitimate decision to use `--network=host` because the user might have an iglu server running on the host machine.  And port 8080 is commonly used in web development, e.g. it is the webpack dev server's default port.

If it fails to open the port, does the docker image fail to start, or is the error silently ignored?  Do you think we gain anything by making the port configurable, e.g. with an optional env variable?

Reply:
Good point, I'm making it possible to override

Reply:
I'm in favour of changing this port, it doesn't really need to be on such a popular port since it's only for the micro pipeline. If making it configurable is too much hassle for now, then something like 8008 (still commonly used for http but less so) or 8088 (not a popular choice for anything) might be a better option, although making it configurable is definitely the optimal solution here.

Reply:
I made it easy to override : https://github.com/snowplow-incubator/snowplow-micro/pull/52/commits/8aba2c9287d11d9a035caac16cdb038413e02ecb